lib/UnoCore/Targets/Android/com/fuse/App.java

```diff
@@ -217,6 +216,11 @@ public void onNewIntent (Intent intent)
             pendingURI = intent.getDataString();
         }
 #endif
+
+#if @(Project.Android.AssociatedDomains:IsSet)
+
+        @(Project.Android.AssociatedDomains:SplitAndJoin('\n\t\t', 'if (intent!=null && intent.getScheme()!=null && intent.getScheme().contains("http") && intent.getDataString().toLowerCase().contains(new String("','").toLowerCase())) { pendingURI = intent.getDataString(); }'))
```

Comment:
Is there a reason for the SplitAndJoin-stuff here? Can't you just put the code in there verbatim instead?

Reply:
Needed to cater for multiple domains in the .unoproj:
```
  "Android": {
    "AssociatedDomains": ["example.com","sub.example.com"],
   ...
```

Reply:
Other syntax ideas for handling multiple?

Reply:
How about generating an array of strings, and avoid repeating the code for each domain?

Reply:
Oh yeah, I guess my mind got stuck in markup land... sorted.